content/academy/pro_apify_scraper_development/apify_sdk.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Apify SDK introduction
+description: Learn about the Apify SDK, as well as some semi-advanced topics which will be important throughout the next lessons in this section of the course.
+menuWeight: 6.1
+paths:
+    - pro-apify-scraper-development/apify-sdk
+---
+
+# [](#the-apify-sdk) The Apify SDK
+
+The SDK is one of the most fundamental parts of developing for the Apify platform. It is an open-source JavaScript library built on top of these technologies:
+
+- Apify actors (which run the Node.js app built with the SDK inside a Docker container)
+- The Apify API/Apify Client (on the Apify platform)
+- Your local operating system (for local development)
+- Puppeteer + Playwright (as headless browser managers)
+- Request + Cheerio (for downloading and parsing HTML)
+
+The SDK factors away and manages the hard parts of the scraping/automation development under the hood, such as:
+
+- Autoscaling
+- Request concurrency
+- Queueing requests
+- Data storage
+- Using and rotating [proxies]({{@link anti_scraping/proxies.md}})
+- Puppeteer/Playwright setup overhead
+- Plus much more!
+
+As a developer, your goal is to take the Apify SDK, which is quite generic on its own, and write website-specific or use-case-specific actors.
+
+## [](#docs) Docs, docs, docs
+
+The Apify SDK and its resources can be found in various different places:
+
+1. [Official documentation](https://sdk.apify.com/)
+2. [Apify SDK guide](https://sdk.apify.com/docs/guides/motivation)
+3. [SDK implementation examples](https://sdk.apify.com/docs/examples/cheerio-crawler)
+4. [Github repository (source code, issues)](https://github.com/apify/apify-js)
+5. [NPM](https://www.npmjs.com/package/apify)
+6. [Help articles](https://help.apify.com/)
+
+## [](#learning) Learning 🧠
+
+Before continuing, it is highly recommended to do the following:
+
+- Look over [the SDK guide](https://sdk.apify.com/docs/guides/motivation) and ideally **code along**
```

Comment:
```suggestion
- Look over [the SDK guide](https://sdk.apify.com/docs/guides/motivation) and, ideally, **code along**.
```

Reply:
Also, please add full stops for each list item. When they're just links like the ones above, it's not needed, but when it's a sentence, we use them